projects/benqi-ignite/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@projects/benqi-ignite",
+    "version": "1.0.0",
+    "scripts": {
+        "yarn": "yarn install",
```

Comment:
Probably better naming could be chosen for install? Or is it some practice for yarn?

Reply:
This one is part of autogenerated project setup. We're not really using it in development, but maybe it has some other usage. I just left it as it is after package generation